scripts/release_processes/README.MD

```diff
@@ -1,5 +1,15 @@
 # Release Processes
 
+## How to update these processes
+
+Have a better idea about how the deployment process should go? Make a PR and it'll be reviewed, just like a code change!
+
+ * Navigate to this directory
+ * Modify the appropriate `.dot` file 
+ * Run `refresh.sh` to update the png files.
```

Comment:
I do not find any file by this name in this PR nor in the repo on `develop`

Reply:
Oops, forgot to add the file. Should be there now.